### PR TITLE
Update golang.org/x/net to latest version to resolve CVE-2022-27664

### PR DESCRIPTION
**Pull request by andytinkham**
### Desired Outcome

Remove versions of golang.org/x/net that are vulnerable to CVE-2022-27664

### Implemented Changes

- Update Go to 1.19
- Upgrade github.com/stretchr/testify, k8s.io/api, and k8s.io/apimachinery to latest versions
- Update replace statements (and add new ones) redirecting golang.org/x/net to v0.0.0-20220923203811-8be639271d50

### Connected Issue/Story

CyberArk internal issue link: CONJSE-1530

### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [X] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [X] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [X] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [X] No behavior was changed with this PR

#### Security

- [X] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [ ] There are no security aspects to these changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +38,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   with Secrets Provider [cyberark/sidecar-injector#76](https://github.com/cyberark/sidecar-injector/pull/76)
 
 ### Security
+- Updated replace statements to force golang.org/x/net v0.0.0-20220923203811-8be639271d50
+   and updated testify to 1.8.0 [cyberark/sidecar-injector#77](https://github.com/cyberark/sidecar-injector/pull/78)
```
Comment (rpothier) [inline CHANGELOG.md:42]: should this be [cyberark/sidecar-injector#78]

**Review by rpothier — CHANGES_REQUESTED** [review]
LGTM! - one minor comment.

File: CHANGELOG.md (hunk 1 above)
Comment (andytinkham) [reply CHANGELOG.md:42]: Yep. should be. Missed one place. Thanks!

**Review by rpothier — APPROVED** [review]
LGTM!

**Pull request merged**